### PR TITLE
Update docs.tuleap.org theme to FontAwesome 5

### PR DESCRIPTION
**Pull request by LeSuisse**
Part of [request #17110](https://tuleap.net/plugins/tracker/?aid=17110): Update to FontAwesome 5

### CODE REVIEW

File: languages/en/_themes/tuleap_org/style/backbone/_comparison-table.scss [hunk 1]
```diff
@@ -327,7 +327,7 @@ $table-border-radius: 6px;
                 color: $tuleap-enterprise;
             }
 
-            > .fa {
+            > .fas, > .fas, > .fab {
```
Comment (Hyzual) [inline languages/en/_themes/tuleap_org/style/backbone/_comparison-table.scss:330]:
```suggestion
            > .fas, > .far, > .fab {
```

File: languages/en/_themes/tuleap_org/style/backbone/_comparison-table.scss [hunk 2]
```diff
@@ -533,8 +533,8 @@ $table-border-radius: 6px;
                 > a {
                     color: $tuleap-enterprise;
 
-                    > .fa {
-                        font-size: 14px;
+                    > .fas, > .fas, > .fab {
```
Comment (Hyzual) [inline languages/en/_themes/tuleap_org/style/backbone/_comparison-table.scss:536]:
```suggestion
                    > .fas, > .far, > .fab {
```

File: languages/en/_themes/tuleap_org/style/backbone/_form.scss [hunk 3]
```diff
@@ -183,7 +183,7 @@ select {
         font-size: 13px;
         line-height: 18px;
 
-        > .fa {
+        > .fas, > .fas, > .fab {
```
Comment (Hyzual) [inline languages/en/_themes/tuleap_org/style/backbone/_form.scss:186]:
```suggestion
        > .fas, > .far, > .fab {
```

File: languages/en/_themes/tuleap_org/style/backbone/_comparison-table.scss [hunk 4]
```diff
@@ -546,8 +546,8 @@ $table-border-radius: 6px;
                 > a {
                     color: $mytuleap;
 
-                    > .fa {
-                        font-size: 14px;
+                    > .fas, > .fas, > .fab {
```
Comment (Hyzual) [inline languages/en/_themes/tuleap_org/style/backbone/_comparison-table.scss:549]:
```suggestion
                    > .fas, > .far, > .fab {
```

File: languages/en/_themes/tuleap_org/style/backbone/_form.scss [hunk 5]
```diff
@@ -270,12 +270,12 @@ select {
     }
 
     &.bounce {
-        > .fa {
+        > .fas, > .fas, > .fab {
```
Comment (Hyzual) [inline languages/en/_themes/tuleap_org/style/backbone/_form.scss:273]:
```suggestion
        > .fas, > .far, > .fab {
```

File: languages/en/_themes/tuleap_org/style/backbone/_form.scss [hunk 6]
```diff
@@ -270,12 +270,12 @@ select {
...
             animation: 500ms ease-in-out bounce;
         }
     }
 
-    > .fa {
+    > .fas, > .fas, > .fab {
```
Comment (Hyzual) [inline languages/en/_themes/tuleap_org/style/backbone/_form.scss:278]:
```suggestion
    > .fas, > .far, > .fab {
```

File: languages/en/_themes/tuleap_org/style/backbone/_step.scss [hunk 7]
```diff
@@ -21,7 +21,7 @@
     align-self: flex-start;
     margin: 0 0 30px;
 
-    > .fa {
+    > .fas, > .fas, > .fab {
```
Comment (Hyzual) [inline languages/en/_themes/tuleap_org/style/backbone/_step.scss:24]:
```suggestion
    > .fas, > .far, > .fab {
```

**Review by Hyzual — CHANGES_REQUESTED** [review]
Thanks for the changes ! It looks ok on my side, although I must admit I have no idea where most of the icons are. I imagine they only exist on tuleap.org and not in the documentation :man_shrugging:

**Comment by LeSuisse** [comment]
> Thanks for the changes ! It looks ok on my side, although I must admit I have no idea where most of the icons are. I imagine they only exist on tuleap.org and not in the documentation man_shrugging

Yes they are used in docs.tuleap.org theme. They are none in the documentation itself.

**Review by Hyzual — APPROVED** [review]

**Pull request merged**